docs/install/openshift.md

```diff
@@ -0,0 +1,263 @@
+## Requirements
+
+Before proceeding, please ensure that you have an OpenShift cluster running K8s
+1.19+ (OpenShift 4.7+) and have Helm 3.5+ installed. In addition, you'll need to
+install the OpenShift CLI (`oc`) to authenticate to your cluster and create OpenShift
+resources.
+
+You'll also want to install the [latest version of Coder](https://github.com/coder/coder/releases/latest)
+locally in order to log in and manage templates.
+
+## Install Coder with OpenShift
+
+### 1. Authenticate to OpenShift and create a Coder project
+
+Run the following command to login to your OpenShift cluster:
+
+```console
+oc login <cluster-url>
+```
+
+This will configure your local `~/.kube/config` file with the cluster credentials
+needed when installing Coder via `helm`.
+
+Next, you will run the below command to create a project for Coder:
+
+```console
+oc new-project coder
+```
+
+### 2. Configure SecurityContext values
+
+Depending upon your configured Security Context Constraints (SCC), you'll need to set
+the following `securityContext` values in the Coder Helm chart:
```

Comment:
I noticed we didn't do this for our internal Openshift deployment. I'm assuming some deployments need it? 

Should we make it clear it's not a required step? And where does one find their `<project-specific UID>?`

Reply:
@bpmct Cian added this in the BuildConfig section.

```
oc get project coder -o json | jq -r '.metadata.annotations'
```

do you think we should add this here as well?

Reply:
Yep!

Reply:
In my install, we didn't have to do any of the seccompProfile, allowPriviledgeEscalation, and capability stuff. It was just

```yaml
coder:
  replicaCount: 1
  env:
    - name: CODER_PG_CONNECTION_URL
      valueFrom:
        secretKeyRef:
          key: url
          name: coder-db-url
    - name: CODER_CACHE_DIRECTORY
      value: /tmp/coder-cache/
  securityContext:
    runAsGroup: 1001080000
    runAsUser: 1001080000
  service:
    enable: true
    externalTrafficPolicy: Cluster
    sessionAffinity: ClientIP
    type: ClusterIP
```

Are there other installs that require all of the other stuff?

Also, I'm noticing that setting different sections of the helm values are different steps. What do you think about combining it into a single step and using comments or an explainer around why each section (securityContext, service type) is important. It might be a bit confusing with multiple steps on all the values that need to be set.

Reply:
@bpmct good feedback, i'll remove those values and also combine the Helm steps (Coder service, access URL, postgreSQL, cache stuff).